Sources/Controllers/ControllerFactory.swift

```diff
@@ -1,50 +1,50 @@
 import Util
+import Responses
 import Requests
 
 public class ControllerFactory {
 
-  public static func getController(_ request: Request) throws -> Controller {
+  public static func process(_ request: Request) throws -> HTTPResponse {
     let publicDir = try CommandLineReader().publicDirectoryArgs()
     let contents = try FileReader(at: publicDir ?? "").read()
-    LogsController.updateLogs(request)
+    LogsController.update(request)
     let path = request.path
 
     switch path {
```

Comment:
My linter gives me this error for this switch statement:
`Function should have complexity 10 or less: currently complexity equals 11`

Where complexity refers to the different control flow paths that the code can take. I was under the impression that having a lot of paths with only one point of comparison is the benefit that switch statements provide.

Is this something I should try to watch in the future?